test/integrationTest.js

```diff
@@ -46,7 +46,7 @@ test('balance', async (t) => {
   })
 })
 
-test('promotion', async (t) => {
+test.skip('promotion', async (t) => {
```

Comment:
perhaps open a follow-up to re-enable this test so it's not forgotten

Reply:
Agree